packages/eslint-plugin-fbtee/src/__tests__/no-empty-strings-test.ts

```diff
@@ -0,0 +1,197 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```

Comment:
Most object keys are sorted already. Why disable it in this file?

fyi, I have VS Code setup to auto save as I type, and then`cmd+s` applies fixes so I just write my objects however I want and they are sorted automatically when I hit `cmd+s`. See https://cpojer.net/posts/the-perfect-development-environment#vs-code

Reply:
For this particular case I thought it made sense to have valid tests before invalid ones, but we can change if you prefer 

Nice, I'll check it out! I have prettier formatting on save but not eslint fixes atm, it caused some unexpected changes sometimes with hook dependencies